specs/subresourceintegrity/index.html

```diff
@@ -146,7 +146,7 @@ <h2 id="introduction">Introduction</h2>
         integrity="sha256-C6CB9UYIS9UJeqinPHWTHVqh/E1uhG5Twh+Y5qFQmYg="&gt;
 </code></pre>
 
-  <p class="example highlight">Scripts, of course, are not the only resource type which would benefit
```

Comment:
Oh, please excuse this worthless comment ;) It seems that this attribute is not associated with any extra styling anyway. Let's go ahead and merge!